packages/react-composites/src/composites/MeetingComposite/state/MeetingAdapterState.ts

```diff
@@ -71,3 +35,58 @@ export interface MeetingAdapterClientState extends Pick<CallAdapterClientState,
  * combined into one to suit the purpose of a Meeting.
  */
 export interface MeetingAdapterState extends MeetingAdapterUiState, MeetingAdapterClientState {}
+
+export function generateMeetingAdapterState(callAdapter: CallAdapter, chatAdapter: ChatAdapter): MeetingAdapterState {
```

Comment:
Recommend changing the signature to:

    mergeAdapterStates(callAdapterState: CallClientState: chatAdapterState: ChatAdapterState): MeetingAdapterState

This function is a state merging function. The calls to `getState()` don't fit with the rest of the responsibility of this function (imagine writing unittests in isolation for this function).

Reply:
`mergeAdapterStates` would imply a full, unbiased merge of the two states - whereas this is being picky and creating a meeting state from call and chat states.

Though you're right its not creating or generating something new - just marshalling existing state data into a format that works for meetings.

Maybe `marshallMeetingState(callAdapter: CallAdapter, chatAdapter: ChatAdapter): MeetingAdapterState`?

Reply:
Maybe `meetingAdapterStateFromCallAndChatAdapterStates` but that's so long

Reply:
Updated to `meetingAdapterStateFromBackingStates` for now - LMK if there's something better